<launch>

  <node name="gnss_odom_node" pkg="gnss_odom" type="gnss_odom" output="screen">
    <param name="cmd_vel_topic" value="/aristos/wheel/cmd_vel" />
    <param name="fix_topic" value="/aristos/filter/positionlla/gnss" />
    <param name="odom_pub_topic" value="gnss_heading/odom" />
    <param name="velocity_threshold" value="0.15"/>
    <param name="distance_threshold" value="0.2"/>
    <param name="heading_offset" value="0.0"/>
  </node>

</launch>